klog.go

```diff
@@ -432,6 +433,7 @@ func InitFlags(flagset *flag.FlagSet) {
 	flagset.Var(&logging.verbosity, "v", "number for the log level verbosity")
 	flagset.BoolVar(&logging.addDirHeader, "add_dir_header", logging.addDirHeader, "If true, adds the file directory to the header of the log messages")
 	flagset.BoolVar(&logging.skipHeaders, "skip_headers", logging.skipHeaders, "If true, avoid header prefixes in the log messages")
+	flagset.BoolVar(&logging.skipPropagation, "skip_propagation", logging.skipPropagation, "If true, avoid propagation of log message to lower severe levels")
```

Comment:
This help could be clearer IMO - it's specifically about writing to multiple files, right?

Reply:
>This help could be clearer IMO

Agreed, I had troubles finding the best wording to describe this, any suggestions?

>it's specifically about writing to multiple files, right?

Not exactly, I did see this happening in terminal output as well.

Reply:
The name and help-text are obscure to me.  The way I would think about this is more like

`--one_output   # If true, only write logs to their native severity level (vs also writing to each lower severity level)`

or 

`--minimize_writes  # If true, only write logs to their native severity level (vs also writing to each lower severity level)`

I think the default behavior is strange and surprising to many people, so writing this in terms of the "obviously normal" behavior sounds good to me.

That said, I don't care strongly, and I won't fight hard :)

Reply:
Thanks for the input, I really like the `one_output` so I'll use that one instead.